.github/workflows/task-checklist.yml

```diff
@@ -0,0 +1,15 @@
+name: Check all tasks are completed
+
+on:
+  check_run:
```

Comment:
Do you want `check_run`…?  It doesn't guarantee a PR exists, you might get errors because `payload.pull_request` won't exist.

Reply:
hmm yeah you're right, i was misunderstanding what `check_run` actually did